src/NuGet.Services.EndToEnd/Support/TestData/E2E.DotnetTool/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
```

Comment:
Header of .net foundation?

Reply:
This is test content copied from the symbols package test data. This is not code compiled into the test DLL. It is actually just test data.